### DISCUSSION TITLE
Is it feasible to decrease (slow down) the speed which Update Manager downloads package files?

### DISCUSSION THREAD

**Discussion by Yekutiel** [linuxmint/discussions#166] [Ideas]
I originally posted the following  on [Linux Mint Forums](https://forums.linuxmint.com/viewtopic.php?p=2375925#p2375925).

On Linux Mint 21.2 Cinnamon I have Update Manager configured to automatically update my system. It works great, yet when it downloads large updates my internet speed temporarily drops significantly. I find this annoying.

Therefore, I'd like to configure Update Manager to download files at, say, 10% of its default speed. As a result, I presume I would be less likely to experience temporarily internet laginess when Update Manager automatically updates my computer.

As you probably surmised, whether my computer were to download a particular update in, say, 2 minutes or 20 minutes, is irrelevant to me. However, experiencing 2 minutes of lagginess while Update Manager automatically updates my computer is a small hassle which I'd rather avoid.

**Comment by 100savage** [comment]
Have you tried to change your mirror to a slower bandwidth?
You can find the mirrors in the update manager/edit/software sources

**Comment by Yekutiel** [comment]
No, the idea of clogging up the internet with this kludge seems wasteful to me.
